src/main/flight/pid.h

```diff
@@ -69,6 +63,14 @@ typedef enum {
     PID_CRASH_RECOVERY_BEEP
 } pidCrashRecovery_e;
 
+typedef enum {
+    HARDFLEX = 0, //error
```

Comment:
nah.

Reply:
hardflex is iterm relax based on pid error, unicorn is based on setpoint, and juicy is based on gyro. I don't see how that is misleading???

Reply:
as long as you guys maintain the code it might be funny.

Reply:
* it is only temporary naming for the purpose of "blind testing".
* not intended to merge as-is.

Reply:
@nerdCopter pull requests have a purpose. they are made to be merged. or at least commented on by the community. why dont you work on branches and keep off PRs when you want to have that placebo thing going?